client/src/components/Form/Elements/FormDrilldown/FormDrilldown.vue

```diff
@@ -0,0 +1,95 @@
+<script setup lang="ts">
+import { computed, type ComputedRef } from "vue";
+import FormDrilldownList from "./FormDrilldownList.vue";
+import { getAllValues, type Option, type Value } from "./utilities";
+
+export interface FormDrilldownProps {
```

Comment:
we probably don't need to `export` this here, if it's not being imported elsewhere